2_ThreeSum.cpp

```diff
@@ -62,31 +62,31 @@ using namespace std;
 vector<vector<int> > threeSum(vector<int> &num)
 {
 	//sort the array
-	sort(num.begin(), num.end());
+    sort(num.begin(), num.end());
     //ret contains the resultant triplets
     vector<vector<int> > res;
     //tuple format of the result
     vector<int> tuples(3);
     for (int i = 0; i < num.size(); ++i) 
-	{
+    {
       // ignore duplicate elements
     	if (i > 0 && num[i] == num[i-1])
     		continue;
       	int target = -num[i];

```

Comment:
I think this variable makes things slightly less readable, to me checking if (num[i] + num[start] + num[end] == 0) makes it more clear what you are trying to do

Reply:
Okay, I'll correct this.